src/style.css

```diff
@@ -0,0 +1,90 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Heavy.ttf");
+  font-style: bold;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Book.ttf");
+  font-style: normal;
+}
+
+html {
+  font-family: "Avenir", "Roboto", sans-serif;
+  font-size: 14px;
+  font-weight: 300;
+}
+
+body {
+  margin: 0;
+}
+
+.form {
+  margin: 22px 7px;
+}
+
+.search__bar {
+  padding-left: 62px;
+  box-sizing: border-box;
+
+  font-size: 22px;
+
+  width: 100%;
+  height: 67px;
+
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
```

Comment:
Don't repeat yourself. Move common styles to `search__bar`  and add smth like `search__bar--small` and `search__bar--big` for styles that are different.